### PR TITLE
featured 360 route

### PR DESCRIPTION
**Pull request by gmarinov**

### CODE REVIEW

**Comment by kschzt** [comment]
what is this @gmarinov ?

File: controllers/threesixty.js [hunk 1]
```diff
@@ -18,3 +18,15 @@ exports.index = function(req, res) {
 	});
 }
 
+exports.featured = function(req,res) {
+	var featuredGraph = '/data/graph/eesn/flamingofront.json'
+	switch (process.env.SITE) {
+		case '360.vizor.io':
+		case 'rc.vizor.io':
+		case 'vizor.io':
```
Comment (kschzt) [inline controllers/threesixty.js]: this sets the same featuredGraph on all sites?

Comment (gmarinov) [inline controllers/threesixty.js]: it sets a live one on the live sites, and leaves a dummy one for the test ones.

**Comment by kschzt** [comment]
lgtm

**Pull request merged**